src/styles/recipe/_recipe.scss

```
@@ -1,7 +1,7 @@
-.card{
+.recipe{
     text-align: justify;
     border: solid 1px;
-    border-color: #f2f2f2;
+    border-color: $border-color;
     font-size: 14px;
```

```suggestion
    font-size: $font-size-default;
```

А еще лучше у body указать     font-size: $font-size-default; и нигде его не переопределять сециально